src/adeira.dev/static/css/custom.css

```diff
@@ -1,76 +1,29 @@
-/* your custom css */
-
 .projectTitle {
   text-align: center;
-  font-weight: 200;
-  color: #fff;
 }
 
-.projectTitle > small {
-  font-weight: 200;
+.projectTitle h4 {
+  margin-bottom: 0;
 }
 
 .pluginRowBlock {
   display: flex;
   justify-content: center;
 }
 
-.buttonWrapper {
-  margin: 0 .5em;
-}
-
 .homeContainer {
-  background-color: #30b8ba;
-  padding: 64px;
-}
-
-.inverseButton {
-  border: 1px solid #fff;
-  color: #fff;
-}
-
-.iconContainer {
-  margin-bottom: 30px;
-  margin-right: 30px;
-  max-width: 200px;
-  float: left;
+  background-color: #55e0c6;
+  padding: 40px;
+  color: #000;
```

Comment:
Should we have used the same black as the rest of the project here?

`var(--ifm-font-color-base)`

And, it doesn't change to white when I switch to dark mode, or is that intentional?

Reply:
It's intentional not to change to color. The background is too bright for doing so - it looked weird.

Reply:
> The background is too bright for doing so - it looked weird.

👍